js_modules/dagster-ui/packages/ui-core/package.json

```diff
@@ -136,7 +136,7 @@
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-storybook": "^0.6.14",
     "faker": "5.5.3",
-    "graphql-codegen-typescript-mock-data": "^3.3.0",
+    "graphql-codegen-typescript-mock-data": "^3.7.1",
```

Comment:
Updated this while trying to figure out why the fragment mocking was broken in JobMetadata.test. It wasn't related (see note below) but I kept the package update anyway.